.github/labeler.yml

```diff
@@ -0,0 +1,4 @@
+# This is used by .github/workflows/pr_review.yml to apply the `pull_ready`
+# label to any root change when it has approval.
+pull_ready:
+- '*'
```

Comment:
You probably want `'**'` instead. From https://github.com/actions/labeler, a single star appears to only match files in the root directory of the repo when we want it to match any changed path.

Also note that this requires the `dot=true` option in the job configuration.

```
apply-pull-ready-label:
  ...
  steps:
    - name: ...
      uses: actions/labeler@...
      with:
        dot: true
```